extension/surveys/survey-generator.js

```diff
@@ -542,11 +542,12 @@ function getFormValues(questionArr, form) {
       // Checkboxes may have multiple answers.
       var answerList = document.querySelectorAll(
           'input[name="' + questionLookup + '"]:checked');
-      for (var j = 0; j < answerList.length; j++) {
-        var response = new SurveySubmission.Response(
-            questionStr, answerList[j].value || 'NOANSWER');
-        responses.push(response);
+      var answer = answerList[0].value || 'NOANSWER';
+      for (var j = 1; j < answerList.length; j++) {
+        answer = answer.concat(",", answerList[j].value)
       }
+      var response = new SurveySubmission.Response(questionStr, answer);
+      responses.push(response);
```

Comment:
this change LGTM, but it makes me realize: what's happening to "Other" answers here? are we completely losing them in the responses (both before and after this change)?